### PR TITLE
Adding wedlocks

### PR DESCRIPTION
**Pull request by julien51**
Wedlocks is our email sending application.
This PR just adds the "core" skeleton with no logic at this point.

Refs #2110

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [X] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
One thing that I was thinking is maybe adding typescript in there too, but I am not sure I understand the setup at this point, so I'll wait for @cnasc to make recos on how to achieve this!

File: wedlocks/src/server.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+import handler from './handler'
+
+http.createServer((req, res) => {
+  handler({}, {}, (response) => {
+    res.writeHead(200, {'Content-Type': 'text/plain'});
+    res.end(response);
+  })
+}).listen(1337, '127.0.0.1');
+
+console.log('Server running at http://127.0.0.1:1337/');
```
Comment (benwerd) [inline wedlocks/src/server.js]: 👀

File: wedlocks/static/index.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<script>
+window.location = "https://unlock-protocol.com/"
+</script>
```
Comment (benwerd) [inline wedlocks/static/index.html]: ☝️

File: wedlocks/src/__tests__/handler.test.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import handler from '../handler'
+
+describe('handler', () => {
+  it('should yield hello world', (done) => {
+    handler({}, {}, (message) => {
+      expect(message).toBe('Hello World')
+      done()
+    })
+  })
+})
```
Comment (benwerd) [inline wedlocks/src/__tests__/handler.test.js]: 👀

**Comment by julien51** [comment]
lol! Yes, of course we can talk about the name! I thought wedlocks was cool... but of course *I* am not that cool, so maybe you have a better idea ;)

**Pull request merged**